<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralInfoWidget</class>
 <widget class="QWidget" name="GeneralInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1">
   <item row="0" column="0">
    <widget class="QFrame" name="frameTitle">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelStatusIcon">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;‚ö™&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="labelTitle">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>–ù–∞–∑–≤–∞ –ê–ó–°, –¢–µ—Ä–º—ñ–Ω–∞–ª 0000</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAddress">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;üìç –ê–¥—Ä–µ—Å–∞: –ó–∞–≤–∞–Ω—Ç–∞–∂–µ–Ω–Ω—è...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPhone">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;üìû –¢–µ–ª–µ—Ñ–æ–Ω: –ó–∞–≤–∞–Ω—Ç–∞–∂–µ–Ω–Ω—è...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="scrollAreaWorplaces">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>68</width>
         <height>435</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frameToos">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="toolButtonInfo">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>–ó–∞–≥–∞–ª—å–Ω–∞ —ñ–Ω—Ñ–æ—Ä–º–∞—Ü—ñ—è (–î–∞—à–±–æ—Ä–¥)</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: transparent;
    border: none;
    border-bottom: 2px solid transparent; /* –ü—Ä–∏—Ö–æ–≤–∞–Ω–∞ –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É */
    padding: 5px;
}

QToolButton:hover {
    background-color: #f1f3f4; /* –õ–µ–≥–∫–∏–π —Ñ–æ–Ω –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–Ω—ñ */
}

QToolButton:checked {
    border-bottom: 2px solid #1a73e8; /* –°–∏–Ω—è –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É –¥–ª—è –∞–∫—Ç–∏–≤–Ω–æ—ó –∫–Ω–æ–ø–∫–∏ */
    color: #1a73e8; /* –°–∏–Ω—ñ–π —Ç–µ–∫—Å—Ç */
    font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/res/Images/information.png</normaloff>:/res/Images/information.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTools</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonRRO">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>–Ü–Ω—Ñ–æ—Ä–º–∞—Ü—ñ—è –ø—Ä–æ –†–†–û</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: transparent;
    border: none;
    border-bottom: 2px solid transparent; /* –ü—Ä–∏—Ö–æ–≤–∞–Ω–∞ –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É */
    padding: 5px;
}

QToolButton:hover {
    background-color: #f1f3f4; /* –õ–µ–≥–∫–∏–π —Ñ–æ–Ω –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–Ω—ñ */
}

QToolButton:checked {
    border-bottom: 2px solid #1a73e8; /* –°–∏–Ω—è –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É –¥–ª—è –∞–∫—Ç–∏–≤–Ω–æ—ó –∫–Ω–æ–ø–∫–∏ */
    color: #1a73e8; /* –°–∏–Ω—ñ–π —Ç–µ–∫—Å—Ç */
    font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/res/Images/RRO_icon.png</normaloff>:/res/Images/RRO_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTools</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonPRK">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>–ö–æ–Ω—Ñ—ñ–≥—É—Ä–∞—Ü—ñ—è –ü–†–ö</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: transparent;
    border: none;
    border-bottom: 2px solid transparent; /* –ü—Ä–∏—Ö–æ–≤–∞–Ω–∞ –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É */
    padding: 5px;
}

QToolButton:hover {
    background-color: #f1f3f4; /* –õ–µ–≥–∫–∏–π —Ñ–æ–Ω –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–Ω—ñ */
}

QToolButton:checked {
    border-bottom: 2px solid #1a73e8; /* –°–∏–Ω—è –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É –¥–ª—è –∞–∫—Ç–∏–≤–Ω–æ—ó –∫–Ω–æ–ø–∫–∏ */
    color: #1a73e8; /* –°–∏–Ω—ñ–π —Ç–µ–∫—Å—Ç */
    font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/res/Images/dispenser.png</normaloff>:/res/Images/dispenser.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTools</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonTanks">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>–Ü–Ω—Ñ–æ—Ä–º–∞—Ü—ñ—è –ø—Ä–æ —Ä–µ–∑–µ—Ä–≤—É–∞—Ä–∏</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: transparent;
    border: none;
    border-bottom: 2px solid transparent; /* –ü—Ä–∏—Ö–æ–≤–∞–Ω–∞ –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É */
    padding: 5px;
}

QToolButton:hover {
    background-color: #f1f3f4; /* –õ–µ–≥–∫–∏–π —Ñ–æ–Ω –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–Ω—ñ */
}

QToolButton:checked {
    border-bottom: 2px solid #1a73e8; /* –°–∏–Ω—è –ª—ñ–Ω—ñ—è –∑–Ω–∏–∑—É –¥–ª—è –∞–∫—Ç–∏–≤–Ω–æ—ó –∫–Ω–æ–ø–∫–∏ */
    color: #1a73e8; /* –°–∏–Ω—ñ–π —Ç–µ–∫—Å—Ç */
    font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/res/Images/tanks.png</normaloff>:/res/Images/tanks.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTools</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidgetInfo">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageSummary">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>84</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QLabel" name="labelLogoInfo">
             <property name="minimumSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../resource.qrc">:/res/Images/gandalf3.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">color: rgb(85, 85, 127);</string>
             </property>
             <property name="text">
              <string>–ó–∞–≥–∞–ª—å–Ω–∞ —ñ–Ω—Ñ–æ—Ä–º–∞—Ü—ñ—è.
–û–±–µ—Ä—ñ—Ç—å —Ä–æ–∑–¥—ñ–ª –Ω–∞ –ø–∞–Ω–µ–ª—ñ —ñ–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç—ñ–≤</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageRRO">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="widgetRROHeader" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="labelTotalRRO">
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;üßæ –ó–Ω–∞–π–¥–µ–Ω–æ –∫–∞—Å–æ–≤–∏—Ö –∞–ø–∞—Ä–∞—Ç—ñ–≤: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonCopyAllRRO">
                <property name="text">
                 <string>–ö–æ–ø—ñ—é–≤–∞—Ç–∏ –í–°–Ü –†–†–û</string>
                </property>
                <property name="icon">
                 <iconset resource="../resource.qrc">
                  <normaloff>:/res/Images/clipboard.png</normaloff>:/res/Images/clipboard.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollAreaRRO">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContentsRRO">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>834</width>
                <height>303</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="layoutRROCards">
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>282</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pagePRK"/>
         <widget class="QWidget" name="pageTanks">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QWidget" name="widgetTanksHeader" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="labelTotalTanks">
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;img src=':/res/Images/tanks.png' width='18' height='18' align='middle'&gt; –†–µ–∑–µ—Ä–≤—É–∞—Ä—ñ–≤: 0.</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>540</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="btnCopyTanks">
                <property name="text">
                 <string>üìã –ö–æ–ø—ñ—é–≤–∞—Ç–∏ —Ç–∞–±–ª–∏—Ü—é</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidgetTanks">
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>–ü–∞–ª—å–Ω–µ</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Max –æ–±'—î–º</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Min –æ–±'—î–º</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Max —Ä—ñ–≤–Ω–æ–º—ñ—Ä</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Mix —Ä—ñ–≤–Ω–æ–º—ñ—Ä</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>–¢—Ä—É–±–æ–ø—Ä–æ–≤—ñ–¥</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupTools"/>
 </buttongroups>
</ui>
